### ISSUE TITLE
Props removed when compiling to Solid

### ISSUE THREAD

**Issue by Acidic9** [BuilderIO/jsx-lite#98]
When compiling to solid js, the props argument from the function seems to get removed.

This can be easily reproduced in the jsx-lite playground itself:
```jsx
export default function MyComponent(props) {
  return (
    <div css={{ padding: '10px' }}>
      <h1>{props.title}</h1>
    </div>
  );
}
```
Ouput to Solid:
```jsx
export default function MyComponent() {
  const state = createMutable({});

  return (
    <div class={css({ padding: "10px" })}>
      <h1 css={{}}>{props.title}</h1>
    </div>
  );
}
```
As you can see, props are removed from the arguments in the output.

**Comment by steve8708** [comment]
Fixed over in #99

**Comment by steve8708** [comment]
thanks for reporting @Acidic9 - output should be much cleaner now:

Input:
```tsx
export default function MyComponent(props) {
  return (
    <div css={{ padding: '10px' }}>
      <h1>{props.title}</h1>
    </div>
  );
}
```

Output:

```tsx
export default function MyComponent(props) {
  return (
    <div class={css({ padding: "10px" })}>
      <h1>{props.title}</h1>
    </div>
  );
}
```
